spec/asar-spec.ts

```diff
@@ -1528,7 +1528,7 @@ describe('asar package', function () {
       originalFs.createReadStream(path.join(asarDir, 'a.asar'));
     });
 
-    itremote('can recursively delete a directory with an asar file in itremote', () => {
+    itremote('can recursively delete a directory with an asar file in itremote using rmdirSync', () => {
```

Comment:
```suggestion
    itremote('can recursively delete a directory with an asar file in it using rmdirSync', () => {
```

I know it was the existing title, but it reads like that may have originally been an autocompletion typo ('it' -> 'itremote')? Odd for the test name to reference `itremote` and it reads like it should say 'with an asar file in it'. Same change needed in the title of the new test.